vocode/streaming/utils/worker.py

```diff
@@ -56,20 +56,26 @@ def start(self) -> None:
         return self.worker_task
 
     async def _forward_to_thread(self):
-        while True:
-            item = await self.input_queue.get()
-            self.input_janus_queue.async_q.put_nowait(item)
+        try:
```

Comment:
👍 


Were you able to run locally `ThreadAsyncWorker`? Is it working as expected?